src/pages/en/guides/cms/storyblok.md

```diff
@@ -185,6 +185,11 @@ export default defineConfig({
 });
 ```
 
+:::caution
+  Your components should exactly match your Storyblok Blok library "technical name" field. If these don't match, or you're trying to reference a component that doesn't exist in Storyblok you'll get an error.
+  :::
```

Comment:
I think we can move this to line 165. Right after the docs explain what they `key` and `value` are.
```suggestion
:::caution
The `key` should exactly match your Blok name in Storyblok to be referenced correctly. If these don't match, or you're trying to reference a component that doesn't exist in Storyblok you'll get an error.
:::
```